projects/assets-library/src/images/image-type.ts

```diff
@@ -0,0 +1,12 @@
+export const enum ImagesAssetPath {
+  ErrorPage = 'assets/images/error-page.svg',
+  LoaderSmallCircle = 'assets/images/loader-small.gif',
+  LoaderLargeSquare = 'assets/images/loader-large.gif',
+  LoaderHorizontal = 'assets/images/loader-horizontal.gif'
+}
+
+export const enum LoaderTypes {
+  SmallCircle = 'small-circle',
+  Horizontal = 'horizontal',
+  LargeSquare = 'large-square'
```

Comment:
Spoke to Michael. 

Let's use smallCircle directly in Spinner. We can remove it from here

Horizontal can be renamed to `ExpandableRow`. LargeSquare can be renamed to 'Page'.

Reply:
Also, we would need to bring back the old svg icon approach for other skeleton loaders.